src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -32,9 +32,9 @@ public class OtlpExporterOptions
 #if NETSTANDARD2_1
         /// <summary>
         /// Gets or sets the target to which the exporter is going to send traces or metrics.
-        /// The valid syntax is described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
+        /// The valid syntax is described at https://github.com/grpc/grpc/blob/master/doc/naming.md. The endpoint value should start with http or https.
```

Comment:
The link to valid syntax is not really applicable for NET2_1, right?

Reply:
that is correct. That naming is for gRPC Core only I think.